packages/p2p/src/components/p2p-empty/p2p-empty.tsx

```diff
@@ -1,10 +1,16 @@
-import classNames from 'classnames';
 import React from 'react';
-import PropTypes from 'prop-types';
+import classNames from 'classnames';
 import { Icon, Text } from '@deriv/components';
-import './p2p-empty.scss';
 
-const P2pEmpty = ({ className, children, has_tabs, icon, title }) => {
+type TP2pEmptyProps = {
+    className?: string;
+    children: React.ReactNode;
+    has_tabs: boolean;
+    icon: string;
+    title: string;
+};
+
+const P2pEmpty = ({ className, children, has_tabs, icon, title }: TP2pEmptyProps) => {
```

Comment:
maybe can make it optional and put default value here `has_tabs=true` or `has_tabs=false`